test/variants/parser/packing08.v3

```diff
@@ -0,0 +1,2 @@
+//@parse
```

Comment:
Can you add some negative parsing tests as well? Also, some simpler cases with only one field, some parts are missing, etc?